packages/events-example/README.md

```diff
@@ -1,59 +1,115 @@
-oclif-hello-world
-=================
-
-oclif example Hello World CLI
-
-[![oclif](https://img.shields.io/badge/cli-oclif-brightgreen.svg)](https://oclif.io)
-[![Version](https://img.shields.io/npm/v/oclif-hello-world.svg)](https://npmjs.org/package/oclif-hello-world)
-[![CircleCI](https://circleci.com/gh/oclif/hello-world/tree/main.svg?style=shield)](https://circleci.com/gh/oclif/hello-world/tree/main)
-[![Downloads/week](https://img.shields.io/npm/dw/oclif-hello-world.svg)](https://npmjs.org/package/oclif-hello-world)
-[![License](https://img.shields.io/npm/l/oclif-hello-world.svg)](https://github.com/oclif/hello-world/blob/main/package.json)
-
-<!-- toc -->
-* [Usage](#usage)
-* [Commands](#commands)
-<!-- tocstop -->
-# Usage
-<!-- usage -->
-```sh-session
-$ npm install -g @theniledev/events-example
-$ nile COMMAND
-running command...
-$ nile (--version)
-@theniledev/events-example/0.15.0 linux-x64 node-v16.16.0
-$ nile --help [COMMAND]
-USAGE
-  $ nile COMMAND
-...
+# Nile Events Example #
+
+This example demonstrates how to use a Nile agent to synchronize your data 
+plane and control plane in real time with Nile events.
+
+Nile doesn't prescribe any particular deployment solution, but here we'll be
+using [Pulumi](https://app.pulumi.com/) to deploy objects into AWS.
+## Prerequisites ##
+
+This example assumes you have:
+
+* [An AWS account](https://aws.amazon.com/free/)
+* [A Pulumi account](https://app.pulumi.com/signup) that's 
+  [connected to your AWS account](https://www.pulumi.com/docs/get-started/aws/begin/)
+* A Nile developer account (see [Appendix](#appendix) for help)
+
+## Configure Your Control Plane ##
+
+1. [Login to the Nile Admin Dashboard](https://nad.thenile.dev/) If you don't 
+  already have one, create a workspace.
+2. Create an entity type named "SkyNet". An simple schema is sufficient for 
+  this example:
+
+```json
+{
+  "name": "SkyNet",
+  "schema": {
+    "type": "object",
+    "properties": {
+      "greeting": {
+        "type": "string"
+      }
+    }
+  }
+}
 ```
-<!-- usagestop -->
-# Commands
-<!-- commands -->
-* [`nile reconcile`](#nile-reconcile)
 
-## `nile reconcile`
+3. Create an organization in your workspace. Name it whatever you'd like.
+4. Create an SkyNet instance in your organization, with a simple greeting:
 
-reconcile nile/pulumi deploys
+```json
+{
+  "greeting": "Come with me if you want to live."
+}
+```
+
+## Configure Your Data Plane ##
+
+You can follow the steps to [create a Pulumi project](https://www.pulumi.com/docs/get-started/aws/create-project/)
```

Comment:
can you actually follow these steps? it is unclear to me how the links to documentation here is related to the commands to run below. I *think* I can skip this and just run the bash commands below. If so, I think re-wording this to be "this is what's happening under the hood and you can ignore it" may make sense.